app/services/builders/second_review_turnaround.rb

```diff
@@ -0,0 +1,18 @@
+module Builders
+  class SecondReviewTurnaround < BaseService
+    def initialize(review)
+      @review = review
+    end
+
+    def call
+      ::SecondReviewTurnaround.create!(
+        review_request_id: @review.review_request_id,
```

Comment:
Could the review not have an associated review request? What would we do in that case?

Reply:
it couldn't, it is a fk,  it will raise an exception

Reply:
Alright, but we should handle that exception, shouldn't we? Because we're not preventing this builder to be called with a review without review request.

Reply:
So no, basically we don't have any review without a review request. We are safe there.